README.md

```diff
@@ -1,9 +1,12 @@
 <img src="figs/logo_01.png" width="150"/>
 
-[![Build Status](https://travis-ci.org/HazyResearch/snorkel.svg?branch=master)](https://travis-ci.org/HazyResearch/snorkel)
-[![Documentation](https://readthedocs.org/projects/snorkel/badge/)](http://snorkel.readthedocs.io/en/master/)
+[![Build Status](https://travis-ci.org/HazyResearch/snorkel.svg?branch=redux)](https://travis-ci.org/HazyResearch/snorkel?branch=redux)
+[![Documentation](https://readthedocs.org/projects/snorkel/badge/?version=redux)](https://snorkel.readthedocs.io/en/master/?badge=redux)
```

Comment:
this should link to the redux branch for now, right? https://snorkel.readthedocs.io/en/redux/